aspnetcore/migration/inc/overview.md

```diff
@@ -0,0 +1,70 @@
+
+If you'd like to skip this over and get started, see [Get started](xref:migration/inc/start).
+
+## App migration to ASP.NET Core
+
+Before starting the migration, the app targets ASP.NET Framework and runs on Windows with its supporting libraries:
+
+![Before starting the migration](~/migration/inc/overview/static/1.png)
+
+Migration starts by introducing a new app based on ASP.NET Core that becomes the entry point. Incoming requests go to the ASP.NET Core app, which either handles the request or proxies the request to the .NET Framework app via [YARP](https://microsoft.github.io/reverse-proxy/). At first, the majority of code providing responses is the .NET Framework app, but the ASP.NET Core app is now set up to start migrating routes:
+
+![start migrating routes](~/migration/inc/overview/static/nop.png)
+
+To migrate business logic that relies on `HttpContext`, the libraries need to be built with `Microsoft.AspNetCore.SystemWebAdapters`. Building the libraries with `Microsoft.AspNetCore.SystemWebAdapters` allows:
+
+* The libraries to be built against .NET Framework, .NET Core, or .NET Standard 2.0.
+* Ensures that the libraries are using APIs that are available on both ASP.NET and ASP.NET Core.
+
+![Microsoft.AspNetCore.SystemWebAdapters](~/migration/inc/overview/static/sys_adapt.png)
+
+<!-- Review: Why does this need to be serialized? Can't one team migrate WebAPI, another specific controllers, another, ASPX pages, etc -->
+At this point, the process migrates routes <!--over one at a time-->. For example, WebAPI or MVC controller action methods, ASPX pages, handlers, or some other implementation of a route. If the route is available in the ASP.NET Core app, it's matched and served.
+
+During the migration process, additional services and infrastructure are identified that must be migrated to run on .NET Core. Options listed in order of maintainability include:
+
+1. Duplicate the code
+2. Link the code in the new project
+3. Move the code to shared libraries
+
+Eventually, the ASP.NET Core app handles more of the routes than the .NET Framework app
+
+![the ASP.NET Core app handles more of the routes](~/migration/inc/overview/static/sys_adapt.png)
+
+During migration, the same route may be available in both the ASP.NET Core and the ASP.NET Framework apps. Duplicate routes allows performing A/B testing to ensure functionality is as expected.
+
+Once the ASP.NET Framework app is no longer needed and deleted:
+
+* The app is running on the ASP.NET Core app stack, but is still using the adapters.
+* The remaining migration work is removing the use of adapters.
+
+![final pic](~/migration/inc/overview/static/final.png)
+
+## System.Web Adapters
+
+The `Microsoft.AspNetCore.SystemWebAdapters` is a collection of runtime helpers that facilitate using old core <!-- Review: What's old core? --> written against `System.Web` while moving to ASP.NET Core.
```

Comment:
Review

Reply:
I think that should be "old code"... maybe just "code"